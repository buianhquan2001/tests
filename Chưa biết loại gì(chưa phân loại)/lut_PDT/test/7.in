10
17 18
      #    #     
      #    #     
      # #  #     
#     # #  # ##  
#     # #  # ##  
#    ## #  # ##  
#    ## #  # ##  
# #  ## #  ####  
###  ## ## ####  
###  ## ## ####  
###  ## ## ####  
###  ## ## ####  
###  ## ## ####  
###  ## ## ####  
###  ## ## ####  
###  ## #######  
###  ## ######## 
#################
20 17
        #          #
        #          #
        #          #
        #          #
   #  # #       #  #
   #  # #       #  #
   #  # #       #  #
   #  # #       #  #
   #  # #  #    #  #
#  #  # # ## #  #  #
#  #  # # ## #  #  #
#  #  # # ## #  #  #
#  #  # # ## #  #  #
#  ## ###### ## #  #
#  ######### ## ## #
# ############# ## #
####################
13 16
   #   #     
   #   #    #
   #   #  # #
#  #   #  # #
#  ##  #  # #
#####  #  # #
#####  #  ###
###### ## ###
###### ## ###
###### ## ###
###### ## ###
###### ######
###### ######
#############
#############
#############
15 19
        #   #  
        ##  #  
 #     ###  #  
 #     ###  #  
 #     ###  #  
##   # ###  ## 
##   # #### ## 
###  # #### ## 
###  # #### ## 
###  # ####### 
###  # ####### 
###  # ####### 
#### # ####### 
#### # ####### 
#### # ########
#### ##########
###############
###############
###############
14 16
     #        
  ## #        
  ## #        
  ## #        
# ## #        
# ## #  ##    
#### #  ##    
#### #  ##    
#### #  ##    
#### #  ##    
######  ### ##
######  ### ##
######  ### ##
######  ### ##
######  ### ##
##############
14 16
            # 
       #   ## 
       #   ## 
       #   ## 
 ##   ##   ## 
 ## # ## #### 
 ## # ## #### 
 ## #### #### 
 ####### #### 
 ####### #### 
 ####### #####
 ####### #####
 ####### #####
 #############
##############
##############
29 11
  #     #    #        #      
# #     ##   #        #  # # 
# # #   ###  #       ### ### 
### #   ###  #  #  # ### ### 
### #   ###  #  # ## ### ### 
### # # ###  # ## ###### ### 
### ### ### ## ######### ### 
####### ### ## ############# 
#############################
#############################
#############################
27 19
  #        #            #  
  #        #          # #  
  #        #   #   #  # #  
  #     #  #   #   # ## #  
  #     #  #   #   # ####  
 ##     #  #   ##  # ####  
 ##     #  #   ##  # ####  
 ## #   #  #   ##  # ####  
 ## #   #  # # ##  # #### #
 ## #   #  # # ##  # #### #
### #   #  # # ##  # #### #
### #   ## # # ##  # #### #
### #   ## ### ##  # #### #
### #   ## ######  # #### #
### ##  ## ####### # #### #
### ### ############ #### #
####### ###################
###########################
###########################
22 12
      #      #     #  
      ##     #     #  
      ##     #  #  #  
      ##     # ##  #  
     ###     # ##  #  
     ###     # ##  ## 
 #  #### #   # ##  ## 
 #  #### ##  # ##  ## 
 #  #######  # ###### 
##  ########## ###### 
############## ###### 
######################
26 13
                       #  
     #     #  #    #   #  
#    ##    #  #    #   ## 
#    ##    #  #    #  ### 
# #  ##    #  #  # ## ### 
### ### #  #  #  # ## ### 
####### ## ## ## #### ### 
####### ## ## ## #### ### 
########## ## ## #### ####
########## ##### #### ####
########## ##### #########
##########################
##########################
