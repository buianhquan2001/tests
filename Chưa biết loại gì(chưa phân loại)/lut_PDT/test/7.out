      #----#     
      #----#     
      #-#--#     
#-----#-#--#-##  
#-----#-#--#-##  
#----##-#--#-##  
#----##-#--#-##  
#-#--##-#--####  
###--##-##-####  
###--##-##-####  
###--##-##-####  
###--##-##-####  
###--##-##-####  
###--##-##-####  
###--##-##-####  
###--##-#######  
###--##-######## 
#################
85
        #----------#
        #----------#
        #----------#
        #----------#
   #--#-#-------#--#
   #--#-#-------#--#
   #--#-#-------#--#
   #--#-#-------#--#
   #--#-#--#----#--#
#--#--#-#-##-#--#--#
#--#--#-#-##-#--#--#
#--#--#-#-##-#--#--#
#--#--#-#-##-#--#--#
#--##-######-##-#--#
#--#########-##-##-#
#-#############-##-#
####################
158
   #---#     
   #---#----#
   #---#--#-#
#--#---#--#-#
#--##--#--#-#
#####--#--#-#
#####--#--###
######-##-###
######-##-###
######-##-###
######-##-###
######-######
######-######
#############
#############
#############
50
        #---#  
        ##--#  
 #-----###--#  
 #-----###--#  
 #-----###--#  
##---#-###--## 
##---#-####-## 
###--#-####-## 
###--#-####-## 
###--#-####### 
###--#-####### 
###--#-####### 
####-#-####### 
####-#-####### 
####-#-########
####-##########
###############
###############
###############
61
     #        
  ##-#        
  ##-#        
  ##-#        
#-##-#        
#-##-#--##    
####-#--##    
####-#--##    
####-#--##    
####-#--##    
######--###-##
######--###-##
######--###-##
######--###-##
######--###-##
##############
36
            # 
       #---## 
       #---## 
       #---## 
 ##---##---## 
 ##-#-##-#### 
 ##-#-##-#### 
 ##-####-#### 
 #######-#### 
 #######-#### 
 #######-#####
 #######-#####
 #######-#####
 #############
##############
##############
28
  #-----#----#--------#      
#-#-----##---#--------#--#-# 
#-#-#---###--#-------###-### 
###-#---###--#--#--#-###-### 
###-#---###--#--#-##-###-### 
###-#-#-###--#-##-######-### 
###-###-###-##-#########-### 
#######-###-##-############# 
#############################
#############################
#############################
91
  #--------#------------#  
  #--------#----------#-#  
  #--------#---#---#--#-#  
  #-----#--#---#---#-##-#  
  #-----#--#---#---#-####  
 ##-----#--#---##--#-####  
 ##-----#--#---##--#-####  
 ##-#---#--#---##--#-####  
 ##-#---#--#-#-##--#-####-#
 ##-#---#--#-#-##--#-####-#
###-#---#--#-#-##--#-####-#
###-#---##-#-#-##--#-####-#
###-#---##-###-##--#-####-#
###-#---##-######--#-####-#
###-##--##-#######-#-####-#
###-###-############-####-#
#######-###################
###########################
###########################
201
      #------#-----#  
      ##-----#-----#  
      ##-----#--#--#  
      ##-----#-##--#  
     ###-----#-##--#  
     ###-----#-##--## 
 #--####-#---#-##--## 
 #--####-##--#-##--## 
 #--#######--#-###### 
##--##########-###### 
##############-###### 
######################
80
                       #  
     #-----#--#----#---#  
#----##----#--#----#---## 
#----##----#--#----#--### 
#-#--##----#--#--#-##-### 
###-###-#--#--#--#-##-### 
#######-##-##-##-####-### 
#######-##-##-##-####-### 
##########-##-##-####-####
##########-#####-####-####
##########-#####-#########
##########################
##########################
89
